src/Symfony/Component/Asset/Context/NullContext.php

```diff
@@ -18,6 +18,13 @@
  */
 class NullContext implements ContextInterface
 {
+    public function __construct()
+    {
+        if (__CLASS__ !== get_class($this)) {
```

Comment:
As that's not supported by IDEs i don't think that's worth it (without talking about the fact that having an annotation for a keyword supported by php looks weird to me :P).